.github/workflows/docker.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish Docker image to DockerHub
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# https://docs.docker.com/build/ci/github-actions/multi-platform/
+jobs:
+  docker:
+    name: Build Docker Image and Publish (only on push)
+    runs-on: ubuntu-latest
```

Comment:
Should Ubuntu Linux 22.04-LTS be specified?

Reply:
I have also changed other GitHub workflows!